lib/components/Snackbar/SnackbarWrapper.tsx

```diff
@@ -0,0 +1,42 @@
+import { StandalonePlaceholderRegularIcon } from "@deriv/quill-icons/Standalone";
+import { Snackbar } from ".";
+import React, { useState } from "react";
+
+export const SnackbarWrapper = () => {
```

Comment:
it's weird but i couldn't find the place where this component is used..

Reply:
its just a wrapper to pass functions into the component to work for now. For the next task it'll be used.